src/index.js

```diff
@@ -21,3 +11,62 @@ function typeWriter() {
 }
 
 typeWriter();
+
+const cardTitles = document.querySelectorAll('#mentors .card-title');
+const cardTexts = document.querySelectorAll('.text-muted');
+const cardImages = document.querySelectorAll('.img-fluid.rounded-start');
+const descriptions = document.querySelectorAll('.description');
+
+const speakers = [
+  {
+    name: 'Bekjon Ishpulatov',
+    title: 'Software Engineer',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti ipsa nulla quam aspernatur voluptatem, cum repellendus sint aliquam sit sequi?',
+    image: './src/images/mentor1.jpg',
+  },
+  {
+    name: 'Steve Vozniak',
+    title: 'Backend Engineer',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti ipsa nulla quam aspernatur voluptatem, cum repellendus sint aliquam sit sequi?',
+    image: './src/images/mentor2.jpg',
+  },
+  {
+    name: 'Maria Sharapova',
+    title: 'HR Manager',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti ipsa nulla quam aspernatur voluptatem, cum repellendus sint aliquam sit sequi?',
+    image: './src/images/mentor3.jpg',
+  },
+  {
+    name: 'Serena Uilyams',
+    title: 'Frontend Engineer',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti ipsa nulla quam aspernatur voluptatem, cum repellendus sint aliquam sit sequi?',
+    image: './src/images/mentor4.webp',
+  },
+  {
+    name: 'Sherlock Holmes',
+    title: 'Fullstack Engineer',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti ipsa nulla quam aspernatur voluptatem, cum repellendus sint aliquam sit sequi?',
+    image: './src/images/mentor5.jpeg',
+  },
+  {
+    name: 'John Smith',
+    title: 'Software Engineer',
+    desciption:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Corrupti ipsa nulla quam aspernatur voluptatem, cum repellendus sint aliquam sit sequi?',
+    image: './src/images/mentor6.jpeg',
+  },
+];
+
+window.addEventListener('load', () => {
+  speakers.forEach((speaker, index) => {
+    cardTitles[index].textContent = speaker.name;
+    cardTexts[index].textContent = speaker.title;
+    descriptions[index].textContent = speaker.desciption;
+    cardImages[index].src = speaker.image;
+  });
+});
```

Comment:
- [ ] Good job adding your feature speakers, but as previously requested, Feature section on the Home page, the feature speakers are organized in one vertical column, but in the case, there are organized in two vertical columns Kindly Fix it.
![219156672-d599bb56-caec-471d-a207-946fd98209de](https://user-images.githubusercontent.com/69077061/219758132-97add184-5f83-4365-a0f8-f93b8abe9f1b.png)

Reply:
- [ ] Your speakers are still in 2 columns in mobile view and this is really not good for the user's experience. The interface can be improved. You can set your `grid-template-columns` to `1fr` on the mobile view.